src/assets/scss/_input-switch.scss

```diff
@@ -1,322 +1,282 @@
 @use 'variables';
```

Comment:
I don't think you've changed anything in this file, right? I only see code style and indentation change which you should revert. 

Apart from the fact that we should avoid pushing unnecessary changes, this is also changing indentation to 4 spaces which is wrong (we prefer 2).